saltstack/base/salt/prerequisites/usr/bin/ccmv2-helper.sh

```diff
@@ -33,12 +40,14 @@ setup_ccmv2() {
 
   INVERTING_PROXY_URL="$CCM_V2_INVERTING_PROXY_HOST"
 
+  CCM_V2_AGENT_ACCESS_KEY_ID=$(grep '^CCM_V2_AGENT_ACCESS_KEY_ID=' ${ACC_KEY_TMP_FILE} | cut -d'=' -f2)
+
```

Comment:
Should be removed, this is a leftover from the previous implementation. (`ACC_KEY_TMP_FILE` is not even initialized at this point.) Env variable `CCM_V2_AGENT_ACCESS_KEY_ID` is already set earlier in the user data (or retrieved from AWS Secrets Manager).